### PR TITLE
Add GitHub as a sponsor

### PR DESCRIPTION
**Pull request by starbelly**

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -159,9 +162,13 @@ prizes:
     sponsor_url: https://www.appsignal.com/elixir?utm_source=spawnfest_sponsored&utm_medium=web
     image: prizes_2021/appsignal.png
   - name: $500 in swag and Amazon Gift Cards
+    image: prizes_2021/swag-amazon.png
     sponsor: Klarna
     sponsor_url: https://www.klarna.com/
-    image: prizes_2021/swag-amazon.png
+  - name: $500 in credits at the Github Shop
```
Comment (pablocostass) [inline _config.yml:168]:
```suggestion
  - name: $500 in credits at the GitHub Shop
```

**Review by pablocostass — APPROVED** [review]

File: _config.yml [hunk 2]
```diff
@@ -108,6 +108,9 @@ gold_sponsors:
   - name: Klarna
     logo: sponsors_2021/klarna.png
     url: https://www.klarna.com/
+  - name: GitHub 
+    logo: sponsors_2021/github.png 
+    url: https://github.com/about/careers?utm_source=announcement&utm_medium=event&utm_campaign=spawnfest-20210918
```
Comment (paulo-ferraz-oliveira) [inline _config.yml:113]: Hm... could this be `github.com` only?

File: _config.yml [hunk 3]
```diff
@@ -159,9 +162,13 @@ prizes:
     sponsor_url: https://www.appsignal.com/elixir?utm_source=spawnfest_sponsored&utm_medium=web
     image: prizes_2021/appsignal.png
   - name: $500 in swag and Amazon Gift Cards
+    image: prizes_2021/swag-amazon.png
```
Comment (paulo-ferraz-oliveira) [inline _config.yml:165]: Why this change? `image` always seems to come last. It's innocuous, of course, but it seems "on purpose" 🤣

**Comment by paulo-ferraz-oliveira** [comment]
I'm approving pending two minor comments. Feel free to merge, though.

**Review by paulo-ferraz-oliveira — APPROVED** [review]

File: _config.yml (hunk 2 above)
Comment (filipevarjao) [reply _config.yml:113]: I think they want to point specifically to this page to check the engagement coming from the spawnfest page `campaign=spawnfest-20210918`

**Review by filipevarjao — APPROVED** [review]

File: _config.yml (hunk 2 above)
Comment (paulo-ferraz-oliveira) [reply _config.yml:113]: It seems odd, to me, that they'd require to use `careers` in that case, but 🤷‍♂️

File: _config.yml (hunk 3 above)
Comment (starbelly) [reply _config.yml:165]: I think I had accidentally removed it and then put it back :)
